<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Gabriele Savoia , Mattia Savoia</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>711</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Wiki Search Engine</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>811</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="result_Layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QListWidget" name="resultWidgetList">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="info_Layout">
     <item>
      <widget class="QLabel" name="info_search_label">
       <property name="text">
        <string>Time : ... s        |        Results : ...   </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>201</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>809</width>
       <height>199</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="search_query">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Write your query here</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_button">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_search">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>5</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SEARCHING</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="limit_spin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="limit_labe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Limit</string>
      </property>
     </widget>
     <widget class="QLabel" name="weighting_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Weighting</string>
      </property>
     </widget>
     <widget class="QComboBox" name="weighting_combo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="groupCombo">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="groupLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Group</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_boost_labe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Text Boost</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_boost_label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Title Boost</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="text_boost_spin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>68</width>
        <height>24</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="title_boost_spin">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>68</width>
        <height>24</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="query_setting_restore_button">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>150</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Restore All</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="expandedTerms">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="expansion_checkBox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Expansion terms</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="page_rank_checkbox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>169</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Page Rank</string>
      </property>
     </widget>
     <widget class="QPushButton" name="evaluation">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>100</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Evaluation</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>791</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>20</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
